src/components/grid-aware/StatsBlock/StatsBlock.tsx

```diff
@@ -19,17 +19,69 @@ const StatCard = ({ number, secondaryNumber, statement }: StatCardProps) => (
   </div>
 );
 
+type AxisProp = {
+  x: string | number;
+  y: number;
+};
+
+type BarStatProps = {
+  axes: AxisProp[];
+  legend: { title: string; subtitle: string };
+  position: string;
+};
+
+const BarStatCard = ({ axes, legend, position }: BarStatProps) => {
+  const isVertical = position === "vertical";
+  const statNumbers = axes.map((axis) => axis.y);
+  const maxNumber = Math.max(...statNumbers);
+  const fillPixels = statNumbers.map(
+    (number) => `${Math.round((number / maxNumber) * 100)}px`
+  );
+
+  return (
+    <div className={s.card}>
+      <div className={`${s.barStat} ${s[position]}`}>
+        {axes.map(({ x, y }, i) => {
+          const isYMaxNumber = y === maxNumber;
+          const yAxis = y.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+
+          return (
+            <div key={`${x}-${y}`} className={`${s.barItem} ${s[position]}`}>
```

Comment:
React keys are meant to be some kind of identifier that is "sticky," even if the data might change due to other interactions on the page. It's not a huge deal on our app, since this is a static website, but I think using just `x` (or the `label` under my suggested name) would be a more appropriate key, since it's closer to being an "identifier" for that particular bar.